### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by khasizadaj**
Creates basic `libunit.a` and `libft.a` libraries  that can be utilized by test to run. At the moment it only test ft_isint function.

Run these commands to see output.
```bash
make && make test && ./routine.a
```

The output should look like this:
```
[FT_ISINT]
[All chars are digits.] : [OK]
[One character is letter.] : [OK]
```

### CODE REVIEW

File: libunit/tests/ft_isint/00_launcher.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    testlist = NULL;
+    ft_printf("[FT_ISINT]\n");
+    name = ft_strdup("All chars are digits.");
+    new = testlist_new(name, &test_all_chars_are_digits);
+    if (!new)
+        printf("Failed to create new testlist node\n");
+    testlist_add_back(&testlist, new);
+
+    name = ft_strdup("One character is letter.");
```
Comment (khasizadaj) [inline libunit/tests/ft_isint/00_launcher.c]: Name should be protected. Or it shouldn't be allocated and freed in `clear` function.

File: libunit/tests/ft_isint/00_launcher.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    t_testlist *testlist;
+    char        *name;
+
+    testlist = NULL;
+    ft_printf("[FT_ISINT]\n");
+    name = ft_strdup("All chars are digits.");
+    new = testlist_new(name, &test_all_chars_are_digits);
+    if (!new)
+        printf("Failed to create new testlist node\n");
+    testlist_add_back(&testlist, new);
```
Comment (khasizadaj) [inline libunit/tests/ft_isint/00_launcher.c]: We better have one function that is able to create new one if list is null and add to the back. That would be much cleaner.

File: libunit/tests/ft_isint/00_launcher.c (hunk 1 above)
Comment (khasizadaj) [reply libunit/tests/ft_isint/00_launcher.c]: Fixed it.

File: libunit/tests/ft_isint/00_launcher.c (hunk 2 above)
Comment (khasizadaj) [reply libunit/tests/ft_isint/00_launcher.c]: Done.

**Review by LuckyIntegral — APPROVED** [review]

**Pull request merged**